packages/odyssey-react-mui/README.md

```diff
@@ -18,10 +18,10 @@ exact version (maximum safety) or pin the major and minor versions
 
 ## Getting Started
 
-Install the package:
+Install the package and peer dependencies:
 
 ```sh
-$ yarn add @okta/odyssey-react-mui
+$ yarn add @okta/odyssey-react-mui @emotion/react @mui/material @mui/icons-material
```

Comment:
Is it really a good idea to have them install MUI? I think it's best if we give them those libraries ourselves; otherwise, you could run into compatibility problems with later versions of MUI and `odyssey-react-mui`.

Reply:
I would recommend keeping the MUI version in Odyssey. If users have to maintain it security fixes have to be done in each individual repo for the MUI versions and potentially Odyssey as well; we can't just update Odyssey and be done with it.

I know we only have `okta-ui-react` and `sign-in-widget`, but we may have hundreds in the future. All of them would need to update MUI rather than just getting on the latest Odyssey.

On the other hand, it could be problematic to update Odyssey unless you patched your particular base version with the latest changes.